<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VocabularyBoard</class>
 <widget class="QWidget" name="VocabularyBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="input">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>380</y>
     <width>541</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tips">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>460</y>
     <width>531</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>请输入单词后按回车键或确认来完成出题</string>
   </property>
  </widget>
  <widget class="QPushButton" name="B_confirm">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>380</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="B_switch">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>600</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>去闯关</string>
   </property>
  </widget>
  <widget class="QPushButton" name="B_logout">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>600</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>注销</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_hi">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>那谁，你好！</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_count">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>你已经在题库中贡献x个单词</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_count_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>你的出题者等级为 x 级</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_exp">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>目前经验进度为：</string>
   </property>
  </widget>
  <widget class="QLabel" name="tips_6">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>600</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>本次出题累计时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_time">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>600</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>0 min</string>
   </property>
  </widget>
  <widget class="QPushButton" name="B_search">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>600</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
